### ISSUE TITLE
[Feature Request] Ability to un-hide individual recipes from hide-by-wildcard.

### ISSUE THREAD

**Issue by Matakor** [mezz/JustEnoughItems#1194]
Or maybe force allow one recipe over a large amount? I love the fact that the hide feature exists, but it occasionally hides just a little too much. The easiest example would be the OpenBlocks tank. I really only need the recipe for the tank itself, not the 100+ fluids that can be put into it. Easiest way to hide the recipes is to use the wildcard method, but that includes the tank itself. The other way is each individually, but that will take 100+ clicks just to save one recipe.

Basically, I think it would help greatly to be able to hide by wildcard, but show the one recipe that's actually useful.

**Comment by Matakor** [comment]
Would like this to be looked at.

**Comment by mezz** [comment]
I am closing this as a duplicate of https://github.com/mezz/JustEnoughItems/issues/195
I'm currently working on a fix for it though, should be merged today.
